index.js

```diff
@@ -6,83 +6,173 @@
  * Licensed under the MIT license.
  */
 
-module.exports = {
-	sets: {
-		ascii: " A1B\'K2L@CIF/MSP\"E3H9O6R^DJG>NTQ,*5<-U8V.%[$+X!&;:4\\0Z7(_?W]#Y)=",
-		dot6:  "⠀⠁⠂⠃⠄⠅⠆⠇⠈⠉⠊⠋⠌⠍⠎⠏⠐⠑⠒⠓⠔⠕⠖⠗⠘⠙⠚⠛⠜⠝⠞⠟⠠⠡⠢⠣⠤⠥⠦⠧⠨⠩⠪⠫⠬⠭⠮⠯⠰⠱⠲⠳⠴⠵⠶⠷⠸⠹⠺⠻⠼⠽⠾⠿"
-	},
-
-	// converts character to Braille symbol
-	convert: function (character) {
-		var sign = undefined;
+var BRAILLE = {
+    " ": " ",
+    "_": "⠸",
+    "-": "⠤",
+    ",": "⠠",
+    ";": "⠰",
+    ":": "⠱",
+    "!": "⠮",
+    "?": "⠹",
+    ".": "⠨",
+    "(": "⠷",
+    "[": "⠪",
+    "@": "⠈",
+    "*": "⠡",
+    "/": "⠌",
+    "\'": "⠄",
+    "\"": "⠐",
+    "\\": "⠳",
+    "&": "⠯",
+    "%": "⠩",
+    "^": "⠘",
+    "+": "⠬",
+    "<": "⠣",
+    ">": "⠜",
+    "$": "⠫",
+    "0": "⠴",
+    "1": "⠂",
+    "2": "⠆",
+    "3": "⠒",
+    "4": "⠲",
+    "5": "⠢",
+    "6": "⠖",
+    "7": "⠶",
+    "8": "⠦",
+    "9": "⠔",
+    "A": "⠁",
+    "B": "⠃",
+    "C": "⠉",
+    "D": "⠙",
+    "E": "⠑",
+    "F": "⠋",
+    "G": "⠛",
+    "H": "⠓",
+    "I": "⠊",
+    "J": "⠚",
+    "K": "⠅",
+    "L": "⠇",
+    "M": "⠍",
+    "N": "⠝",
+    "O": "⠕",
+    "P": "⠏",
+    "Q": "⠟",
+    "R": "⠗",
+    "S": "⠎",
+    "T": "⠞",
+    "U": "⠥",
+    "V": "⠧",
+    "W": "⠺",
+    "X": "⠭",
+    "Z": "⠵",
+    "]": "⠻",
+    "#": "⠼",
+    "Y": "⠽",
+    ")": "⠾",
+    "=": "⠿"
+  },
 
-		// searches sets
-		for (var i = this.sets.ascii.length - 1; i >= 0; i--) {
-			if(this.sets.ascii[i] == character){
-				sign = this.sets.dot6[i];
-			}
-		}
-		
-		// if not found will return undefined
-		return sign;
-	},
+  ASCII = {
+    " ": " ",
+    "⠸": "_",
+    "⠤": "-",
+    "⠠": ",",
+    "⠰": ";",
+    "⠱": ":",
+    "⠮": "!",
+    "⠹": "?",
+    "⠨": ".",
+    "⠷": "(",
+    "⠪": "[",
+    "⠈": "@",
+    "⠡": "*",
+    "⠌": "/",
+    "⠄": "\'",
+    "⠐": "\"",
+    "⠳": "\\",
+    "⠯": "&",
+    "⠩": "%",
+    "⠘": "^",
+    "⠬": "+",
+    "⠣": "<",
+    "⠜": ">",
+    "⠫": "$",
+    "⠴": "0",
+    "⠂": "1",
+    "⠆": "2",
+    "⠒": "3",
+    "⠲": "4",
+    "⠢": "5",
+    "⠖": "6",
+    "⠶": "7",
+    "⠦": "8",
+    "⠔": "9",
+    "⠁": "A",
+    "⠃": "B",
+    "⠉": "C",
+    "⠙": "D",
+    "⠑": "E",
+    "⠋": "F",
+    "⠛": "G",
+    "⠓": "H",
+    "⠊": "I",
+    "⠚": "J",
+    "⠅": "K",
+    "⠇": "L",
+    "⠍": "M",
+    "⠝": "N",
+    "⠕": "O",
+    "⠏": "P",
+    "⠟": "Q",
+    "⠗": "R",
+    "⠎": "S",
+    "⠞": "T",
+    "⠥": "U",
+    "⠧": "V",
+    "⠺": "W",
+    "⠭": "X",
+    "⠵": "Z",
+    "⠻": "]",
+    "⠼": "#",
+    "⠽": "Y",
+    "⠾": ")",
+    "⠿": "="
+  };
 
-	// converts Braille symbol to letter
-	read: function (symbol) {
-		var letter = undefined;
-
-		// searches sets
-		for (var i = this.sets.ascii.length - 1; i >= 0; i--) {
-			if(this.sets.dot6[i] == symbol){
-				letter = this.sets.ascii[i];
-			}
-		}
-		
-		// if not found will return undefined
-		return letter;
-	},
+module.exports = {
+  convert: function (character) {
+    return !!BRAILLE[character] ? BRAILLE[character] : " ";
```

Comment:
I think `?` will do. There seems to be no question mark in Braille alphabet as well as it is a common practice to use a question mark as replacement for unsupported signs in many applications e.g. MySQL, CLI.

JS will do the auto coertion anyway. No need for double not imo. So I'd stick to what I have previously written i.e. `return BRAILLE[character] || "?";`.